### ISSUE TITLE
Problems to run sdk-cordova-example

### ISSUE THREAD

**Issue by gvmarvelos** [IndoorAtlas/sdk-cordova-examples#19]
Hello I am trying to run the example for a few days, however it has two errors, the first is that when I use the MapBox Token it gives several unresolved GETs.

Failed to load resource: the server responded with a status of 410 (Gone)

And the second error is that I tested it without the mapbox token, however, no image or map appears in my application, even though there is no error and the CurrentStatus is apparently ok.

CurrentStatus
code: 2
message: "Available"
name: "AVAILABLE"
__proto__: Object

A few months ago it was working and today it is no longer. Did you test this example?

**Comment by oseiskar** [comment]
Please confirm the following and reopen the issue after that:
 1. You have succesfully fingerprinted the building in which you are located and verified that positioning works with Map Creator 2 on Android (see https://indooratlas.freshdesk.com/support/solutions/articles/36000079590-indooratlas-positioning-overview). You will not get any sane outputs from this, or the other code examples, unless the fingerprinting has been completed
 2. You have created an API key, which works with the native [Android](https://github.com/IndoorAtlas/android-sdk-examples) or iOS examples
 3. You have a valid MapBox token created on your own MapBox account

**Comment by gvmarvelos** [comment]
Hi.

1 - Yes I did this process, including installing the beacons, doing all the mapping with a cell phone and then the build. As I said earlier, this example has worked in another time. If necessary I pass my account to verify.

2 - Yes, I created the API Key and placed it in the example APIKeys.js file.

3 - Yes my mapbox key is valid, I also tested it without the key, in the example it describes that the use of the mapbox is for outdoor areas. As it is a test, I left without and continue with the same behavior.

If you need a print, I send it here.

Thanks for answering.

**Comment by oseiskar** [comment]
Thank you. To debug the issue, you should print the [trace ID](https://docs.indooratlas.com/cordova/latest/#gettraceid) and check  with the _Session Viewer_ tool on https://app.indooratlas.com if there are some obvious problems in that session.

In 2. I meant that you should check it with the android-sdk-examples or ios-sdk-examples first, to verify that this is indeed a problem with the Cordova plugin or this example Cordova app (this repository), and not the IndoorAtlas Android/iOS SDK itself or something specific to your deployment.

Cordova tools have a lot of bugs & cache issues themselves and sometimes such issues are resolved by doing a clean build, e.g., `cordova platform remove android && cordova platform add android && cordova run`.

If positioning works on the android-sdk-examples or ios-sdk-examples but not this app, please send a trace ID demonstrating the problem to the [support portal](https://indooratlas.freshdesk.com/support/home).

**Comment by gvmarvelos** [comment]
Hello thanks for the suggestions.

I tried to use the traceID in the Session Viewer tool, but it seems to be wrong, nothing appears, just a red line.

I used TraceID directly from the object so I don't know what might have happened.

I used the examples directly from the indoor atlas git.

I used these commands, cordova platform removes android && cordova platform add android && cordova run, however the results were the same.
